template/android/app/proguard-rules.pro

```diff
@@ -9,6 +9,16 @@
 
 # Add any project specific keep options here:
 
-# com.mypackage should match the package value in your app/src/main/AndroidManifest.xml file.
--keep class com.mypackage.BuildConfig { *; }
+# com.projectname should match the package value in your app/src/main/AndroidManifest.xml file.
+-keep class com.projectname.BuildConfig { *; }
 -keepresources string/build_config_package
+
+# Strip logging from production builds
+-assumenosideeffects class android.util.Log {
```

Comment:
Logging can be useful though when we want to debug a production build with Logcat 😕 Is it considered a good practice to strip them?

Reply:
This is something we did in a client project to be compliant with MASVS checklist, maybe @jcouture could enlighten us more?